### PR TITLE
update to ContainerAnalysis GA release

### PR DESCRIPTION
**Pull request by catherinejones**
This PR takes the work we did to support the open source Grafeas implementation, and focuses it on updating Voucher's support for the ContainerAnalysis API GA release.

### CODE REVIEW

File: cmd/config/metadataclient.go [hunk 1]
```diff
@@ -18,10 +18,22 @@ func NewMetadataClient(ctx context.Context) (voucher.MetadataClient, error) {
...
 	}
 
-	return grafeas.NewClient(
-		ctx,
-		viper.GetString("image_project"),
-		viper.GetString("binauth_project"),
-		keyring,
-	)
+	metadataClient := viper.GetString("metadata_client")
+	switch metadataClient {
```
Comment (Nilay-Shah) [inline cmd/config/metadataclient.go]: Why have a case statement If we only support 1 option? I'm thinking maybe we have an `if` statement which will do the `log.Warning` message and continue returning a `containeranalysis.NewClient` anyways?

File: testing/metadata_client.go [hunk 2]
```diff
@@ -68,9 +64,8 @@ func (c *MetadataClient) GetBuildDetails(ctx context.Context, reference referenc
 	return c.details[refString], nil
 }
 
-func (c *MetadataClient) AddAttestationToImage(ctx context.Context, reference reference.Canonical, payload voucher.AttestationPayload) (voucher.MetadataItem, error) {
-	occ := new(voucher.MetadataItem)
-	return *occ, nil
+func (c *MetadataClient) AddAttestationToImage(ctx context.Context, reference reference.Canonical, payload voucher.AttestationPayload) (interface{}, error) {
+	return nil, nil
```
Comment (Nilay-Shah) [inline testing/metadata_client.go]: I'm unsure what this does/provide?

File: cmd/config/metadataclient.go (hunk 1 above)
Comment (catherinejones) [reply cmd/config/metadataclient.go]: We will be supporting more options once the open source Grafeas PR is finished! Originally this switch statement had both grafeas and containeranalysis as options.

File: testing/metadata_client.go (hunk 2 above)
Comment (catherinejones) [reply testing/metadata_client.go]: This ensures that it implements voucher.MetadataClient, which requires this function. We don't have tests which utilize this stub, so we just leave it as empty.

**Pull request merged**